ts/components/screens/ListItemComponent.tsx

```diff
@@ -0,0 +1,106 @@
+import { ListItem, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import customVariables from "../../theme/variables";
+import H5 from "../ui/H5";
+import IconFont from "./../ui/IconFont";
+import { BadgeComponent } from "./BadgeComponent";
+
+type Props = Readonly<{
+  title: string;
+  subTitle: string;
+  onPress: () => void;
+  isFirstItem?: boolean;
+  isLastItem?: boolean;
+  hasBadge?: boolean;
+  iconName?: string;
+  hideIcon?: boolean;
```

Comment:
We need both iconName (optional) and hideIcon?

Reply:
As @matteodesanti said by default the component must be rendered without icon.

Reply:
iconName is optional because only some items whats to render a custom icon instead of the right arrow. I used hideIcon instead of showIcon as default because I notice the most of the item request a icon

Reply:
Do you prefer I revert it adding showIcon attrinute to our lists?